### DISCUSSION TITLE
npm package for email sending

### DISCUSSION THREAD

**Discussion by vsnandanwar** [iTwin/itwinjs-core#5575] [Q&A] [answered]
Hi,

Is there any itwin npm package for implementation of sending notification through email?

Thanks,
Vikram

**Comment by pmconne** [comment]
Can you be more specific about what you're trying to achieve? This doesn't sound like a feature that would belong in iTwin.js.

**Reply by vsnandanwar** [reply]
I have a service built on node.js, I am using itwin packages for calling the rpc implementation. I want to send an email to the users after some operations. I can use open source npm packages but as I am using itwinjs, Is there any npm package to send an email in itwin js?

**Reply by pmconne** [reply, marked as answer]
No, iTwin.js doesn't supply that kind of functionality.
